client/my-sites/navigation/index.jsx

```diff
@@ -49,39 +46,13 @@ class MySitesNavigation extends Component {
 					{ ...asyncProps }
 				/>
 			);
-
-			// For the new Jetpack cloud sidebar, it has its own site picker.
-			renderSitePicker = false;
-
-			sitePickerProps = {
-				showManageSitesButton: false,
-				showHiddenSites: false,
-			};
 		} else if ( this.props.isGlobalSidebarVisible ) {
 			return this.renderGlobalSidebar();
 		} else {
 			asyncSidebar = <AsyncLoad require="calypso/my-sites/sidebar" { ...asyncProps } />;
-
-			sitePickerProps = {
-				showManageSitesButton: true,
-				showHiddenSites: true,
-				maxResults: 50,
-			};
 		}
 
-		return (
-			<div className="my-sites__navigation">
-				{ renderSitePicker && (
-					<SitePicker
-						allSitesPath={ this.props.allSitesPath }
-						siteBasePath={ this.props.siteBasePath }
-						onClose={ this.preventPickerDefault }
```

Comment:
It looks like `preventPickerDefault` is unused after this removal and the function could be removed?

Reply:
Good catch, removed in 7551d085d4f6260079afe1f17d6e93509214fe89.